### PR TITLE
build(examples): dont assume gh-pages branch exists locally

### PR DESCRIPTION
**Pull request by priyajeet**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @priyajeet has signed the CLA. Thanks for the pull request!

File: conf/cut_release.sh [hunk 1]
```diff
@@ -144,7 +144,11 @@ build_examples() {
 
 push_to_gh_pages() {
     printf "${blue}Pushing styleguide to gh-pages...${end}"
-    git branch -D gh-pages || return 1
+    if git branch -D gh-pages; then
```
Comment (booc0mtaco) [inline conf/cut_release.sh]: Might be good to suppress the error message entirely. I think this will still print it but it's not really an error for us

Comment (priyajeet) [reply conf/cut_release.sh]: I'll try `git branch | grep -w "gh-pages"`

**Pull request merged**